test/integration/query-tests/circle-radius/zoom-and-property-function/style.json

```diff
@@ -0,0 +1,71 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "width": 64,
+      "height": 64,
+      "queryGeometry": [
+        32,
+        16
+      ],
+      "ignored": {
+        "native": "https://github.com/mapbox/mapbox-gl-native/issues/6518"
```

Comment:
https://github.com/mapbox/mapbox-gl-native/issues/6518 is "Add support for identity function type" and is closed. Can you open a new parity issue and update this URL (and in the other test too)?